.vscode/settings.json

```diff
@@ -36,4 +36,12 @@
     "quarto"
   ],
   "cSpell.language": "en,en-GB",
+  "cSpell.words": [
+    "Arnqvist",
+    "Elina",
+    "Immonen",
+    "Liljestrand",
+    "mitonuclear",
+    "Rönn"
+  ],
```

Comment:
We need to find a better solution than putting it into the `settings.json` file, but that's a focus for another time.